installer.rb

```diff
@@ -128,6 +134,7 @@
   f.puts "ACTIVE_DAYS=#{active_days}"
   f.puts "JWT_KEY=#{jwt_key}"
   f.puts "ENC_KEY=#{enc_key}"
+  f.puts "WEBHOOK_PASS=#{hook_pass}"
```

Comment:
I'm not super familiar with ruby, what happens when they say no to this above? do they get a null or blank string or something here? Does `hook_pass` not need to be initialized explicitly in ruby?

Reply:
I changed it to initiate as an empty string. 

I also made it so enabling auth is the default on the installer.